### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by ps96068** [jowilf/starlette-admin#587] [Q&A] [answered]
Dear Colleagues,

it is my first experience of using starlette-admin in my project.
my project is in FastAPI, SQLAlchemy 2.

can you guide me with advice and/or ideas how to achieve the following:

I have attached a picture. in the top left is "Welcome Admin", I would like to replace "Admin" with "username" of the current user.
![st_admin](https://github.com/user-attachments/assets/e706ae9f-abf2-48a3-9c53-22cc10b31679)

**Comment by jowilf** [comment, marked as answer]
You can create a base view for all your views and override the [title method](https://jowilf.github.io/starlette-admin/api/views/?h=basev#starlette_admin.views.BaseView.title)

something like
```py
class MyModelBaseView (sqla.ModelView):

      def title(self, request):
              return f"welcome {request.state.user.name}" 
```

**Reply by ps96068** [reply]
thank you very much for answer
